app/views/components/fixedEstablishmentRadio.scala.html

```diff
@@ -0,0 +1,66 @@
+@*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@this(
+    radios: GovukRadios
+)
+
+@(
+        form: Form[_],
+        ukText: Option[String] = None,
+        iomText: Option[String] = None,
+        otherText: Option[String] = None,
+        headingKey: String,
+        hint: Option[Html] = None,
+        headingMessageArgs: Seq[String] = Seq(),
+        isPageHeading: Boolean = true,
+        inline: Boolean = true,
+        classes: String = "govuk-fieldset__legend--xl"
+)(implicit messages: Messages)
+
+@radios(Radios(
+    classes = if (inline) "govuk-radios--inline" else "",
+    idPrefix = None,
+    name = "value",
+    fieldset = Some(Fieldset(
+        legend = Some(Legend(
+            content = Text(messages(headingKey, headingMessageArgs:_*)),
+            isPageHeading = isPageHeading,
+            classes = classes
+        ))
+    )),
+    hint = hint.map { hint => Hint(
+        content =   HtmlContent(hint)
+)},
+    items = Seq(
+        RadioItem(id = Some("value-uk"),
+                value = Some("true"),
+                content = Text(ukText.getOrElse(messages("variantB.site.uk"))),
+                checked = form("value").value.contains("true")
+        ),
+        RadioItem(id = Some("value-iom"),
+            value = Some("true"),
+            content = Text(iomText.getOrElse(messages("variantB.site.iom"))),
+            checked = form("value").value.contains("true")
+        ),
+        RadioItem(id = Some("value-other"),
+            value = Some("false"),
+            content = Text(otherText.getOrElse(messages("variantB.site.other"))),
+            checked = form("value").value.contains("false")
+            )
```

Comment:
The existing inputRadios component allows you to pass in a Seq of RadioItems. Could you use that component to cut down on repeated code or is there a reason you need this bespoke component?